buku.1

```diff
@@ -218,6 +218,9 @@ Show selective monochrome output with specific fields. Works with --print. Searc
 .BI \-j " " \--json
 Output data formatted as json, works with --print output and search results.
 .TP
+.BI "--colors=" COLORS
```

Comment:
Please use the following syntax like it's for other options in this man page:

    .BI \--colors " COLORS"